ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/ExceptionRequestsPage.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import {
+    Flex,
+    FlexItem,
+    PageSection,
+    Tab,
+    TabTitleText,
+    Tabs,
+    Title,
+} from '@patternfly/react-core';
+
+type TabKey = 'PENDING_APPROVALS' | 'APPROVED_DEFERRALS' | 'APPROVED_FALSE_POSITIVES';
```

Comment:
I believe we need a fourth tab for "Denied" as well https://docs.google.com/presentation/d/1PCYmIS9SkRr9lnZdPJ4bdEkFD3cdq2vYZExq0P_ltn4/edit?pli=1#slide=id.g27eff3a2f70_0_12.

From what I can tell in the requirements doc we need to include this page, but the ability to re-open denied requests will be post-MVP.

Reply:
Good point! I didn't even notice the updated slides section. Was wondering why the slides looked unchanged. Will add that as well

Reply:
Yeah unfortunately I think the linked slide is the _only_ slide with the change. 🤷‍♂️

Reply:
This answers a question in my mind: Deferral Management includes both deferrals and false positives.

Reply:
@pedrottimark I think they've decided to rename it to Exception Management now